### PR TITLE
Repogroup page copy edits

### LINKED ISSUE #12057: Edits to search examples pages
- [x] on all pages, please remove the colons from all headers

- [ ] on all pages, please render code terms in code font

https://sourcegraph.com/search

- [x] ~remove the colon from the end of "Diff/commit search keywords:"~ (already fixed)

- [x] ~Change "Structural Searches" to "Structural searches"~ (already fixed)

- [x] I'm not happy with the "Search a language" section. If the languages listed are the search terms, they should be all lowercase. If they're the names of the languages, they need to be properly capitalized (JavaScript, GraphQL, HTML, JSON, TypeScript)
   - Made them all lowercased to match the search terms. See [comment](https://github.com/sourcegraph/sourcegraph/issues/12057#issuecomment-656550640

https://sourcegraph.com/refactor-python2-to-3

- [x] change the main header to read:
The examples below help you find Python 2 code that requires refactoring, and review examples of the new Python 3 syntax, across popular Python repositories. 

- [x] change the "Python 3 explicit relative imports paragraph to
Python 3 requires you to explicitly specify package location relative to the current folder, and deprecates implicit relative reports. The search below returns usages of explicit relative imports. For example, an import with one leading dot signifies that the package resides in the current folder, and with two leading dots tells Python that the package is one directory up from the current one. 

- [x] Change "Python 2 & 3 integer conversion:" to "Python 2 and 3 integer conversion"

https://sourcegraph.com/react-hooks

- [x] Change "Search popular React Hook repositories from resources in the GitHub repository 'rehooks/awesome-react-hooks'. The search examples show usages of the React Hook useState with various data types. useState is a hook that lets you add React state to function components." 
to
"Search popular React Hook repositories from the GitHub repository 'rehooks/awesome-react-hooks'. The search examples show usage of the React Hook `useState` with various data types."

- [x] please change "Javascript" to "JavaScript" in both places it appears on the page

- [x] please change "Typescript" to "TypeScript"

https://sourcegraph.com/kubernetes

- [x] Change "Search the trending and most starred Kubernetes repositories on GitHub. Explore with search examples below." to "Explore Kubernetes repositories on GitHub. Search with examples below." 

- [x] Change "Browse diffs for recent code changes in the community:" to "Browse diffs for recent code changes"

https://sourcegraph.com/golang

- [x] Change "Search the most starred Go repositories on GitHub. Explore with search examples below." to "Use these search examples to explore Go repositories on GitHub."

https://sourcegr12aph.com/android

- [x] change "Search the most starred Android repositories on GitHub. Explore with search examples below." to "Use these search suggestions to explore popular Android repositories on GitHub."

- [x] change "An intent filter is used to specify the type of intents a component would like to receive. An intent filter can accept 3 types of elements - <action>, <category> and <data> elements." to "Intent filters specify the type of intents a component would like to receive. An intent filter can accept three types of elements - <action>, <category> and <data> elements."

### PR DESCRIPTION
**Pull request by christinaforney**
Fixes #12057

### CODE REVIEW

**Comment by felixfbecker** [comment]
This doesn't fix the capitalization issue of the language section, right? (part of #12057)

image

File: web/src/search/input/SearchPage.tsx [hunk 1]
```diff
@@ -194,7 +194,7 @@ export const SearchPage: React.FunctionComponent<Props> = props => {
                                         to={`/search?q=lang:${language.filterName}`}
                                         key={language.name}
                                     >
-                                        {language.name}
+                                        {language.filterName}
```
Comment (christinaforney) [inline web/src/search/input/SearchPage.tsx:197]: > This doesn't fix the capitalization issue of the language section, right? (part of #12057)

@felixfbecker - this change should have fixed the language names to be the lowercased names. Here's what it looks like: 

image

Here's a comment discussing in the original issue: https://github.com/sourcegraph/sourcegraph/issues/12057#issuecomment-656550640

Comment (felixfbecker) [reply web/src/search/input/SearchPage.tsx:197]: With them being lowercase, it also now makes sense to me for them to be monospace!

Comment (christinaforney) [reply web/src/search/input/SearchPage.tsx:197]: Agree! :)

**Pull request merged**